api/trace/api.go

```diff
@@ -43,15 +43,19 @@ type Tracer interface {
 	) error
 }
 
-type EndOptions struct {
+// EndConfig provides options to set properties of span at the time of ending
+// the span.
+type EndConfig struct {
 	EndTime time.Time
 }
 
-type EndOption func(*EndOptions)
+// EndOption applies changes to EndConfig that sets options at span end time.
```

Comment:
perhaps '...sets options when the span is ended' ?

Reply:
Fixed in the branch using committer privileges.